Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Ordering-Pages/index.md

```diff
@@ -1,15 +1,32 @@
 ---
-versionFrom: 7.0.0
-needsV8Update: "true"
+versionFrom: 8.0.0
 ---
 
 # Ordering Pages
+This is intended as an instructional guide for how you can sort your items in the tree section.
 
-The pages in Umbraco are sorted in the tree structure according to a predefined sort order. It is most common for the page that has been created most recently to be placed at the bottom of the tree structure, however, you can change the sort order of the pages in a section of the website by using the sort function.
+The pages in Umbraco are sorted in the tree structure according to a predefined sort order. It is most common for the page that has been created most recently to be placed at the bottom of the tree structure, you can change the ***sort*** order of the pages in a section of the website by using the sort function.
```

Comment:
This sentence is more than 40 characters 🙈 
Could you try making it a tat shorter?